ci/dockerfiles/runtime_functions.sh

```diff
@@ -305,6 +305,14 @@ build_ubuntu_cpu() {
     build_ubuntu_cpu_openblas
 }
 
+build_ubuntu_base_cpu() {
+    echo "skip"
```

Comment:
Do we need this?

Reply:
Yes, otherwise build.py -a doesn't work.

Reply:
Ok, I see